src/inc/quic_trace.h

```diff
@@ -104,6 +104,7 @@ typedef enum QUIC_TRACE_API_TYPE {
     QUIC_TRACE_API_CONNECTION_COMPLETE_RESUMPTION_TICKET_VALIDATION,
     QUIC_TRACE_API_CONNECTION_COMPLETE_CERTIFICATE_VALIDATION,
     QUIC_TRACE_API_STREAM_PROVIDE_RECEIVE_BUFFERS,
+    QUIC_TRACE_API_CONNECTION_POOL_CREATE,
```

Comment:
ETW manifest, windbg and wpa plugins will need to be updated accordingly.

Reply:
I updated the manifest, but several of our preview features are not added to the windbg or WPA plugins. 
I want to add them all together in a later PR so this PR doesn't also contain work for e.g. providing receive buffers.